src/views/pages/Home/Home.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="Agriboard">
```

Comment:
minor but the naming convention for files that we've been using is Home/index.vue. Not sure if we will be needing it for the vue project to be consistent. Maybe ask@Right2Drive  for his opinion